### DISCUSSION TITLE
Link nodes using completions

### DISCUSSION THREAD

**Discussion by Apricot1024** [foambubble/foam#1362] [Ideas]
I have ever used org-roam before, and there is a feature I like. Once I set org-roam-completion-everywhere to t, it can auto complete my typo to a wikilink. 

For example, I have a node with title "Just a normal node". Then,  when I type "Just" or longger, org-roam will give me a completion suggestion. I get the option to complete "Just" to a wikilink which link to node "Just a normal node", and if the suggestion is wrong, I can ignore it and keep noting.

When noting, maybe I have mentioned a certain word which actually refers to a node exists, but I donot realize it. With this feature, I will never miss possible wikilinks, and decide if it is really a wikilink I want by myself.

Would you please add a feature like that.

![image](https://github.com/foambubble/foam/assets/68466174/9b504937-bca7-453b-adce-a138daad138b)

**Comment by riccardoferretti** [comment]
I believe you are describing wikilink completion, which Foam supports for both existing notes (i.e. links to a file) and for placeholders (i.e. dangling wikilinks that do not have a corresponding existing file).
Is that correct?
